crates/api_models/src/payments.rs

```diff
@@ -1258,15 +1258,15 @@ pub enum BankRedirectData {
     },
     Sofort {
         /// The billing details for bank redirection
-        billing_details: BankRedirectBilling,
+        billing_details: Option<BankRedirectBilling>,
 
         /// The country for bank payment
         #[schema(value_type = CountryAlpha2, example = "US")]
-        country: api_enums::CountryAlpha2,
+        country: Option<api_enums::CountryAlpha2>,
 
         /// The preferred language
         #[schema(example = "en")]
-        preferred_language: String,
+        preferred_language: Option<String>,
```

Comment:
why do we have changes for Sofort pm type in this pr? I observed there was another PR for sofort, please undo these changes if not necessary

Reply:
Yes, there is another separate PR for Sofort. Sofort changes appeared in this branch as  `dynamic_field/giropay` branch is built upon `dynamic_field/sofort` branch whilst the PR pointed to main. I have changed the base branch. Can you please review it, again.